po/pl.po

```diff
@@ -1345,39 +1371,39 @@ msgstr "Twardy, kruchy, szarobiały metal"
 
 #: ../libelemental/data.cc:69
 msgid "Italian: manganese, Latin: magnes (magnet)"
-msgstr ""
+msgstr "Włoskie: manganese, łacińskie: magnes (magnes)"
```

Comment:
There should be semicolons rather than commas separating languages -- I know that this is wrong in the original English, but it seems like a good idea to fix it in the translation. There are a few more instances of this.

Reply:
(I think this is optional, though.)